Source/Scene/Scene.js

```diff
@@ -3358,6 +3427,106 @@ define([
         return result;
     }
 
+    var logDepthRegex = /\s+czm_writeLogZ\(/;
+    var extensionRegex = /\s*#extension\s+GL_EXT_frag_depth\s*:\s*enable/;
+
+    function getLogDepthShaderProgram(context, shaderProgram) {
```

Comment:
Should this and `createLogDepthCommand` be in a private utility class in another file?